Regen and update demo

BrainBlasted opened the pull request:
Regenerates the crate with the latest GIR

BrainBlasted committed:
Update submodules

BrainBlasted committed:
Regen and update demo

Regenerates the crate with the latest GIR

bilelmoussaoui commented:
thanks a lot, i will have to update the docker image first

bilelmoussaoui commented:
It should pass the build part, but needs few fixes for the new gtk-rs checker tests

BrainBlasted committed:
Add LICENSE and license headers

BrainBlasted committed:
Fix Gir.toml indents

BrainBlasted committed:
ci: Disable gtk-rs checker

This checker is looking for a license header on an
auto-generated file. This means an unfixable error
that causes CI to fail.

For now, disable the checker.

bilelmoussaoui commented:
Thank you :D
